pleskdistup/common/src/dns.py

```diff
@@ -19,7 +26,13 @@ def get_all_includes_from_bind_config(config_file: str) -> list:
             for line in f:
                 line = line.strip()
                 if line.startswith("include"):
-                    include_file = line.split('"')[1].strip(";")
+                    include_file = line.split('"')[1]
+
+                    if include_file.startswith("/"):
+                        include_file = chroot + include_file
+                    else:
+                        include_file = os.path.join(os.path.dirname(current_file), include_file)
```

Comment:
This is not a correct logic. For relative paths the file will be assumed to be in the directory previously nominated by a `directory` statement. In the absence of a `directory` option the directory from which bind was loaded is used.

Honestly, I see no reason to support this, as I don't think we use this.

Reply:
You right; I assumed 'bind' works differently. Let's just block these types of paths and disregard them. It is unlikely that anyone has this kind of configuration. If they do, the conversion will simply stop due to the missing include file, which isn't a major issue.